docs/source/api/python/nn.pytorch.rst

```diff
@@ -171,6 +171,13 @@ GCN2Conv
     :members: forward
     :show-inheritance:
 
+SignConv
```

Comment:
Perhaps you can change `SignConv` to `SignedConv` to distinguish it from SIGN (SIGN: Scalable Inception Graph Neural Networks).